### PR TITLE
Add "sol compile --output-dir" option

### PR DESCRIPTION
**Pull request by Artem-Zhdanov**
New option was called "--output-dir", as it is called in sol compiler

The following cases have been tested:
```
tondev sol compile Contract.sol
tondev sol compile Contract.sol -o abc               # relative path
tondev sol compile Contract.sol -o /home/ruser  #absolute path

tondev sol compile abc/Contract.sol                   # absolute path
tondev sol compile abc/Contract.sol -o abc
tondev sol compile abc/Contract.sol -o /home/ruser
```

### CODE REVIEW

File: src/controllers/solidity/index.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
     uniqueFilePath,
 } from "../../core/utils";
 import fs from "fs";
+import mv from 'mv';
```
Comment (d3p) [inline src/controllers/solidity/index.ts:13]: Need to add mv node module to package.json

**Review by d3p — APPROVED** [review]

File: src/controllers/solidity/index.ts (hunk 1 above)
Comment (Artem-Zhdanov) [reply src/controllers/solidity/index.ts:13]: Done

**Review by joydark — APPROVED** [review]

File: src/controllers/solidity/index.ts [hunk 2]
```diff
@@ -55,8 +56,14 @@ export const solidityCompileCommand: Command = {
             title: "Source file",
             nameRegExp: /\.sol$/i,
         },
+        {
+            name: 'output-dir',
+            alias: 'o',
+            type: 'folder',
+            title: 'Output folder (current is default)',
```
Comment (melsomino) [inline src/controllers/solidity/index.ts:63]: Need to set defaultValue: "". Otherwise this option is required.

**Review by melsomino — CHANGES_REQUESTED** [review]
see comments

**Pull request merged**